packages/nitro-rpc-client/src/transport/http.ts

```diff
@@ -60,3 +62,12 @@ export class HttpTransport {
     };
   }
 }
+
+// For testing with self-signed certs, ignore certificate errors. DO NOT use in production.
+export function unsecureHttpsAgent(): https.Agent {
+  // For testing with self-signed certs, ignore certificate errors. DO NOT use in production.
```

Comment:
Can we put this inside `internal` or otherwise make it harder to use accidentally?

Reply:
It's `ts` code, so we don't have the option of making it an `internal` thing. It does look unused though, so could be deleted.